frontend/src/pages/userDashboard.tsx

```
@@ -3,152 +3,152 @@ import { useNavigate } from "react-router-dom";
-                View Alerts
-              </button>
+          
+          {showSearchResults && (
+            <div 
+              className="absolute top-12 left-0 w-full bg-white shadow-lg rounded-lg p-2 z-50"
+              onMouseLeave={() => setShowSearchResults(false)}
+            >
+              {isLoading ? (
+                <div className="p-4"><LoadingSpinner /></div>
+              ) : (
+                <ul role="listbox" className="max-h-48 overflow-y-auto">
```

Biome (1.9.4)

[error] The HTML element with the interactive role "listbox" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "listbox".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)